### ISSUE TITLE
'EXGLView' has no propType for native prop 'EXGLView.accessibilityLabel' of native type 'String'

### ISSUE THREAD

**Issue by Josephmaxim** [gre/gl-react#117]
Hi, 
Just finished setting up a fresh react native project and successfully linked all libraries for Andriod and IOS I am having this issue on Android and can't seem to figure out the problem. (Works on ios)
<img width="314" alt="screen shot 2017-05-29 at 5 58 07 pm" src="https://cloud.githubusercontent.com/assets/10732770/26562859/6bb995b6-4498-11e7-8fc3-0916edaf90aa.png">

**Comment by gre** [comment]
Which version of RN? I have an idea why it might happen

**Comment by Josephmaxim** [comment]
@gre 

> `"dependencies": {
    "gl-react": "^3.6.0",
    "gl-react-native": "^3.6.0",
    "react": "16.0.0-alpha.6",
    "react-native": "0.44.2",
    "react-native-fetch-blob": "^0.10.5",
    "react-navigation": "^1.0.0-beta.10",
    "unsplash-js": "^4.6.0"
  }`

**Comment by Josephmaxim** [comment]
Hi @gre ,
It looks like the linking/installation steps are outdated. Compared to the examples provided, I noticed that running `react-native link` will also link the gl-react-native lib. commenting it prevents it from build failed errors. but I get the `'EXGLView' has no propType for native prop 'EXGLView.accessibilityLabel' of native type 'String'` error and the app crashes.

**android/settings.gradle**
<img width="877" alt="screen shot 2017-05-31 at 5 20 15 pm" src="https://cloud.githubusercontent.com/assets/10732770/26654693/c61de582-4625-11e7-8f8a-fbe4908f999c.png">

**android/app/build.gradle**
<img width="608" alt="screen shot 2017-05-31 at 5 20 50 pm" src="https://cloud.githubusercontent.com/assets/10732770/26654896/88df1258-4626-11e7-8d24-427accfce2e0.png">

**android/build.gradle**
<img width="668" alt="screen shot 2017-05-31 at 5 21 04 pm" src="https://cloud.githubusercontent.com/assets/10732770/26654989/d0fa7bfe-4626-11e7-8aea-8baaf9be4dac.png">

**MainApplication.java**
<img width="570" alt="screen shot 2017-05-31 at 5 21 32 pm" src="https://cloud.githubusercontent.com/assets/10732770/26655031/0018de44-4627-11e7-96b1-d2555a708e65.png">

Uncommenting the gl-react-native will cause a build error of:
<img width="471" alt="screen shot 2017-05-31 at 5 34 26 pm" src="https://cloud.githubusercontent.com/assets/10732770/26655128/707a785a-4627-11e7-831c-8bbc67c57137.png">

**Comment by kingerking** [comment]
Yeah, ive been getting the same error

**Comment by gre** [comment]
(i would be happy to accept PR for this, but i'm currently away for a few weeks so not very active at the moment.)

it appears that RN had a breaking changes around the View.propTypes , but i have not investigated much. I think we need to synchronise some code from EXGLView from the Expo repo. they have changed a bunch of stuff so i'm not sure how easy it will be but also need to integrate the stuff about the Camera support.

**Comment by kelset** [comment]
Been facing the same issue today, using `RN 0.44.3`.
Tried to investigate a bit around, and modify the code inside the `node_module/gl-react-native` locally hoping to find a way to help out. Couldn't find a solution atm, but I think this StackOverflow [answer](https://stackoverflow.com/a/34244587/1473216) could provide a direction to aim for.

Btw I think @Josephmaxim issue may be that re-running react-native link actually added the line `import fr.greweb.rngl.RNGLPackage;` twice in the `MainApplication.java` file, before the the `public class MainApplication` definition.

(you can find the test project I'm trying to build on Android [here](https://github.com/kelset/react-native-glshaders-testing))

**Comment by martykan** [comment]
Found the solution, will submit PR soon.

**Comment by kelset** [comment]
Btw just noticed, I think we can close this issue since it gets fixed by commit https://github.com/gre/gl-react/commit/0b502ce51b783f973e8c8680ab010221a5b030ad. @gre any plans to do a release for `gl-react-native`? (can't figure out how to make `yarn` point to the master of it)

**Comment by gre** [comment]
yeah i'll try to release a minor tomorrow, i wish to finish support of RN 0.45 too
